src/transformV2.ts

```diff
@@ -142,8 +143,8 @@ export function transformV2(options: TransformV2Options): TemplateV2Options {
 
   const direction = options.direction || 'ltr';
   const templateOptions: BaseTemplateOptions = {
-    intro: new SafeString(marked(options.intro, { renderer })),
-    outro: new SafeString(marked(options.outro, { renderer })),
+    intro: new SafeString(xss(marked(options.intro, { renderer }))),
+    outro: new SafeString(xss(marked(options.outro, { renderer }))),
     question: options.question.label,
```

Comment:
@jsedlacek do we want to sanitise question as well?

Reply:
@ksvitkovsky question is not a SafeString, thus automatically escaped by Handlebars.